tetris.py

```diff
@@ -187,6 +188,23 @@ def new_game():
     tetris_control.game = game
 
 
+def start_dialogue():
```

Comment:
Consider renaming this. A function named `start_foo()` gives me the impression that the function ends before `foo` completes. But this function waits for the dialogue to complete, because that's how `mb.askokcancel` works.